fenix/app/src/androidTest/java/org/mozilla/fenix/onboarding/view/OnboardingMapperTest.kt

```diff
@@ -71,12 +75,30 @@ class OnboardingMapperTest {
         )
     }
 
+    @Test
+    fun showDefaultBrowserPageFalse_showNotificationFalse_showAddWidgetTrue_pagesToDisplay_returnsSortedListOfAllConvertedPages_withoutNotificationPage() {
```

Comment:
We should add a test that asserts that caption is actually present on the first page always, regardless of the combinations

Reply:
Good idea, I'll add some tests to make that happen.